CHANGELOG/v1.6.3.md

```diff
@@ -0,0 +1,38 @@
+## 👌 Kubernetes version support
+
+- Management Cluster: v1.25.x -> v1.29.x
+- Workload Cluster: v1.23.x -> v1.29.x
+
+[More information about version support can be found here](https://cluster-api.sigs.k8s.io/reference/versions.html)
+
+## Changes since v1.6.2
+## :chart_with_upwards_trend: Overview
+- 12 new commits merged
+- 2 bugs fixed 🐛
+
+## :bug: Bug Fixes
+- CAPD: Fix(capd): remove hack for btrfs/zfs support (#10192)
+- e2e: Test: retry GetOwnerGraph in owner references test on certificate errors (#10217)
+
+## :seedling: Others
+- CABPK: Add pod metadata to capbk manager (#10212)
+- clusterctl: Add 0 default to worker-machine-count help (#10205)
+- clusterctl: Add hivelocity infra provider to clusterctl (#10168)
+- Dependency: Bump go version to 1.21.8 (#10246)
+- Release: Improve release-staging build (#10190)
+- Release: Read in dockerfiles from stdin (#10209)
+
+:book: Additionally, there have been 3 contributions to our documentation and book. (#10230, #10241, #10243) 
+
+## Dependencies
```

Comment:
Should we drop this or add a "No dependency changes in this release" statement?

Reply:
Hm. Q: Why is this section empty? We have a bump in l.22

Reply:
this was updated with the force push just a few minutes ago: https://github.com/kubernetes-sigs/cluster-api/pull/10252/files#r1521783375

Reply:
does that look right you @sbueringer ?

Reply:
All good. Sorry I think I missed the sub sections :)